ZFitter/script/Init_calibration_procedure.sh

```diff
@@ -11,15 +11,15 @@ fi
 #add branch r9 => do not use it until the r9 corrections are available
 ##./script/addBranch.sh data/validation/${file}.dat R9Eleprime
 #make pileupHist: they are used to make pileupTrees, and also they are required for step1
-pileupHist
+##pileupHist
```

Comment:
True, probably it's worth adding a check "if the histogram already exists, skips this part" and a "force statement" via command line